src/main/java/io/appform/functionmetrics/FunctionTimerAspect.java

```diff
@@ -171,11 +187,9 @@ private void updateTimer(Timer timer, Stopwatch stopwatch) {
     private String convertToString(Object obj) {
         if (obj == null) {
             return "";
-        }
-        if (obj instanceof String) {
+        } else if (obj instanceof String) {
```

Comment:
else on next lines like rest of the code

Reply:
Done